src/styles/brackets.less

```diff
@@ -1392,11 +1372,11 @@ a, img {
     font-size: 14px;
     color: @bc-text;
     background: @bc-panel-bg;
-    overflow: visible;  // needed for .error popup
+    overflow: visible; // needed for .error popup
     padding: 5px 4px 4px 14px;
 
     -webkit-transform: translate(0, 0); // Prefix still required.
-    transition: -webkit-transform 66ms cubic-bezier(0, 0.62, 0.04, 0.99);
+    transition: transform 66ms cubic-bezier(0, 0.62, 0.04, 0.99);
```

Comment:
If we are still using the unprefixed version of `transform` we will need both transition versions here, and the unprefixed transform:

```
-webkit-transform: translate(0, 0); // Prefix still required.
transform: translate(0, 0);
transition: -webkit-transform 66ms cubic-bezier(0, 0.62, 0.04, 0.99);
transition: transform 66ms cubic-bezier(0, 0.62, 0.04, 0.99);
```

Same with the other uses of transform and transitions with transforms.